src/models/GameCharacter.ts

```diff
@@ -4,9 +4,14 @@ export interface IGameCharacter extends mongoose.Document{
     name: string,
     level: number,
     healthLevel: number,
-    owner: string,
     power: number,
     attackSpeed: number,
+    ConnectGameCharacterWithUser
+    skills: Array,
+    User: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "User"
+    }
```

Comment:
Interface should have `userId: string` field instead of User object reference

Reply:
done